### DISCUSSION TITLE
欢迎来到 ZY Player 交流区

### DISCUSSION THREAD

**Discussion by Hunlongyu** [Hunlongyu/ZY-Player#378] [交流]
## 👋 Welcome!

在这里我们可以尽情交流讨论. 我们也会在这里发布新闻公告. Enjoy ~

**Comment by hfby** [comment]
ZYAPP能不能加个投屏功能，想大屏幕看，开发者们辛苦了！

**Reply by ssjlzg** [reply]
ZYAPP能不能加个密码锁避免出现尴尬场面，开发者们辛苦了！

**Comment by LJX20161** [comment]
开发大大你好，我导入直播源了以后去检测可用性的时候很慢很慢，有些是可以连上但是很卡，能不能加一个超时之类的去优化一下谢谢

**Reply by LJX20161** [reply]
还有就是在播放器界面，3秒过后频道列表会自动消失，但是我再想点开却找不到入口

**Comment by dzhsmc** [comment]
厚着脸皮希望出个TV版

**Comment by dacidacida** [comment]
你好，我电脑用zy播放有点小卡100兆宽带都带不动，缓存太慢了

**Comment by Jun128682613** [comment]
你好，有没有最新的视频源

**Comment by qishuai** [comment]
你好收到您的邮件。我会尽快给您回复。谢谢，保持联系！
